renderer/components/UI/Button.js

```diff
@@ -10,6 +10,7 @@ const Wrapper = styled(BaseButton)`
   font-weight: 300;
   line-height: '18px';
   white-space: nowrap;
+  text-align: ${({ justify }) => justify};
```

Comment:
maybe call this prop `textAlign` ? sounds a bit confusing since `justify` is a prop name for `text-align` in css

Reply:
Ok got it